spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -245,14 +284,34 @@ class Recurrent[T : ClassTag](var batchNormParams: BatchNormParams[T] = null)
     // Clone N modules along the sequence dimension.
     initHidden(outputSize.drop(2))
     cloneCells()
+    if (maskZero) {
```

Comment:
add requirement to appoint that input should be 3 dimensions if using  "maskZero"